### DISCUSSION TITLE
Bulk import DCIM Site

### DISCUSSION THREAD

**Discussion by neharaina1771** [netbox-community/netbox#15834] [Help Wanted!]
Hi ,

Can I bulk import the sites along with region using import feature.
We have region which has different parent but same child region.How can I bulk import Site based on parent region

**Comment by candlerb** [comment]
> Can I bulk import the sites along with region using import feature.

Yes. Import the regions first (if they're not already imported), then import the sites.

> We have region which has different parent but same child region.

Sorry, I don't understand what you mean.

In the data model, a region has a single parent. A region can have multiple children (all those with their parent equal to this one), but a region cannot be the child of multiple parents.

That is: regions form a tree, not a graph.

You can use SiteGroups as another way to categorise sites: e.g. use Regions for geographical areas and SiteGroups as a functional classification. SiteGroups form their own independent tree.

> How can I bulk import Site based on parent region

Each site is linked to one particular region, so you just specify that region as a column in the site import. A site can belong to a region at any level of the tree. Whether that region is also a a child of another region, or a parent of other regions, is irrelevant.
